src/net.c

```diff
@@ -307,13 +307,19 @@ static unsigned int psk_server_callback(SSL *ssl, const char *identity, unsigned
 }
 #endif
 
+void ssl_keylog_callback_func(const SSL *ssl, const char *line)
```

Comment:
With a few exceptions, TLS is used instead of SSL throughout the project. Would you please change this to `tls_` and make similar changes in the few other places where SSL is used.

Please also guard this function inside a `#ifdef WITH_TLS` block and check it compiles when using `make WITH_TLS=no`.